packages/components/src/components/text/text.jsx

```diff
@@ -1,27 +1,27 @@
 import classNames from 'classnames';
 import React from 'react';
 
-const Text = ({ children, size, color, align, weight, lineHeight, as, className }) => {
-    function setStyle() {
+const Text = ({ children, size, color, align, weight, line_height, as, className, ...props }) => {
+    const setStyle = () => {
         const style = {
             '--text-size': `var(--text-size-${size || 's'})`,
             '--text-color': `var(--text-${color || 'general'})`,
-            '--text-lh': `var(--text-lh-${lineHeight || (as === 'p' ? 'm' : 'xs')})`,
+            '--text-lh': `var(--text-lh-${line_height || 'm'})`,
             '--text-weight': `var(--text-weight-${weight || 'normal'})`,
         };
         return style;
-    }
-    function setClassName() {
+    };
+    const setClassName = () => {
         const classStyle = ['dc-text', `${align ? `dc-text--${align}` : ''}`, className];
         return classStyle;
-    }
+    };
     return (
         (as === 'p' && (
-            <p className={classNames(setClassName())} style={setStyle()}>
+            <p {...props} className={classNames(setClassName())} style={setStyle()}>
                 {children}
             </p>
         )) || (
-            <span className={classNames(setClassName())} style={setStyle()}>
+            <span {...props} title={props.title} className={classNames(setClassName())} style={setStyle()}>
```

Comment:
Also I think we prefer out tool-tips over HTML default ones. So better to skip `title` here. Right?

Reply:
Same question for `setStyle`.